### PR TITLE
Adding HC4 bindings and pre-built indexes

### PR DESCRIPTION
**Pull request by ToluClassics**

### CODE REVIEW

File: pyserini/resources/index-metadata/lucene-index.hc4-v1.0-chinese.20220714.cd2601.README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Lucene index for HC4 v1.0 (Chinese).
+
+This index was generated on 2022/07/14 at Anserini commit [`cd2601`](https://github.com/castorini/anserini/commit/cd26013fe6f1a8bcaebc440392e6c97c7bd486b7) on `orca` with the following command:
+
+```
+target/appassembler/bin/IndexCollection -collection NeuClirCollection \
+  -generator DefaultLuceneDocumentGenerator -threads 8 \
+  -input /store/collections/multilingual/hc4-v1.0-zh/ \
+  -index indexes/lucene-index.hc4-v1.0-chinese.20220714.cd2601/ \
+  -storePositions -storeDocvectors -storeRaw -optimize -language zh \
```
Comment (lintool) [inline pyserini/resources/index-metadata/lucene-index.hc4-v1.0-chinese.20220714.cd2601.README.md:12]: Remove trailing slash?

Comment (lintool) [reply pyserini/resources/index-metadata/lucene-index.hc4-v1.0-chinese.20220714.cd2601.README.md:12]: same below also?

**Review by lintool — APPROVED** [review]

**Comment by lintool** [comment]
Oh, add test cases please:
https://github.com/castorini/pyserini/blob/master/tests/test_load_qrels.py
https://github.com/castorini/pyserini/blob/master/tests/test_load_topics.py

**Comment by lintool** [comment]
lgtm

**Comment by ToluClassics** [comment]
Hi @lintool, I don't have write access to merge the PR

**Pull request merged**